Update Clock to drive all bits of connected signal (#2290)
cocotb/cocotb: documentation/source/newsfragments/2290.change.rst

> In :class:`cocotb.clock.Clock`, drive all bits of the connected signal.

Reviewer: ```suggestion
In :class:`cocotb.clock.Clock`, when given a vector signal, all bits of the signal are driven.
```

> When passing a vector signal to :class:`cocotb.clock.Clock`, drive all bits of the signal, not just the right-most bit.

Reviewer 2: ```suggestion
:class:`cocotb.clock.Clock` now drives all bits of a vector signal.
```
